services/core-web/src/components/mine/Permit/ViewPermit.tsx

```diff
@@ -72,10 +63,10 @@ const ViewPermit = () => {
           </Title>
         </Col>
         <Col>
-          <CoreTag icon={<FontAwesomeIcon icon={faLocationDot} />} text={mine?.mine_name} />
```

Comment:
Looks like it was actually intended to introduce this new tag style after all 😅 

<img width="1072" alt="image" src="https://github.com/user-attachments/assets/6339e295-e074-481f-892a-
d9b10b80e6c8">